plutus-core/src/Language/PlutusCore/Evaluation/Machine/Cek.hs

```diff
@@ -376,7 +353,7 @@ computeCek ctx _ (Constant ex val) =
 -- s ; ρ ▻ builtin bn  ↦  s ◅ builtin bn arity arity [] [] ρ
 computeCek ctx env (Builtin ex bn) = do
     -- TODO: budget?
-  arity <- arityOf bn
+  BuiltinRuntime _ arity _ _ <- ask >>= lookupBuiltin bn . cekEnvRuntime
```

Comment:
> Honestly, I can't even remember the precedence of `>>=` and  `.`

- `>>=` has one of the lowest precedences. Pretty much everything apart from `$` has the same or a higher precedence (even `&`)
- `.` has one of the highest precedences, Pretty much everything apart from function application has the same or a lower precedence (even ``id . id `const` id`` fails to parse, because `` `const` `` has the same precedence as `.`)
 
So these two operators are very far apart from each other.

I can make it

```haskell
  BuiltinRuntime _ arity _ _ <- lookupBuiltin bn . cekEnvRuntime =<< ask
```

Or we can have an `asksM` combinator and write

```haskell
  BuiltinRuntime _ arity _ _ <- asksM $ lookupBuiltin bn . cekEnvRuntime
```

But I'm rather used to writing `_ <- _ >>= _` so I probably have that in many places.

Reply:
So I'm changing

```haskell
BuiltinRuntime _ arity _ _ <- ask >>= lookupBuiltin bn . cekEnvRuntime
```

to

```haskell
    BuiltinRuntime _ arity _ _ <- asksM $ lookupBuiltin bn . ckEnvRuntime
```

and don't feel like I'm making any noticeable change at all (apart from now using a definition that is not in `Control.Monad.Reader`).

Anyway, either way is fine for me.

Reply:
Done.